### PR TITLE
[windcentrale] Add default translations

### PR DESCRIPTION
**Pull request by marcelrv**
+eliminate remaining sat warnings

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.windcentrale/src/main/resources/OH-INF/i18n/windcentrale.properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+channel-type.windcentrale.runPercentage.label = Run Percentage
+channel-type.windcentrale.runPercentage.description = Run percentage this year
+channel-type.windcentrale.runTime.label = Run Time
+channel-type.windcentrale.runTime.description = Run time this year
+channel-type.windcentrale.timestamp.label = Last Updated
+channel-type.windcentrale.windDirection.label = Wind Direction
+channel-type.windcentrale.windSpeed.label = Wind Speed
+
+# Status messages
+offline.mill-data-error=Failed to process mill data
```
Comment (wborn) [inline bundles/org.openhab.binding.windcentrale/src/main/resources/OH-INF/i18n/windcentrale.properties:44]:
```suggestion
# status messages

offline.mill-data-error = Failed to process mill data
```

File: bundles/org.openhab.binding.windcentrale/src/main/resources/OH-INF/i18n/windcentrale_nl.properties [hunk 2]
```diff
@@ -23,3 +34,6 @@ channel.windcentrale.runPercentage.description = Molen productie percentage dit
 channel.windcentrale.runTime.label = Molen productietijd
 channel.windcentrale.runTime.description = Molen productietijd dit jaar
 channel.windcentrale.timestamp.label = Ververs tijdstip
+
+# Status messages
+offline.mill-data-error=Fout bij het verwerken van de molen data
```
Comment (wborn) [inline bundles/org.openhab.binding.windcentrale/src/main/resources/OH-INF/i18n/windcentrale_nl.properties:39]:
```suggestion
# status messages

offline.mill-data-error = Fout bij het verwerken van de molen data
```

**Review by wborn — APPROVED** [review]
Thanks! I also had this on my todo list. :wink:

**Comment by marcelrv** [comment]
great! thx

**Pull request merged**